Fastfile

```diff
@@ -307,7 +272,7 @@ reflected in `Info.plist` during building."
       app_identifier: bundle_identifier,
       force: false,
       p12_password: p12_password,
-      output_path: pem_output_path % build_configuration.to_s
+      output_path: "#{build_configuration.to_s}_push_certificates"
```

Comment:
I don't like this change. I prefer a constant that can be easily identified and changed.

Reply:
I agree.

Please check this commit, where I extracted the `output_path` to a property inside the lane:

https://github.com/Wolox/fastlane-mobile/commit/fc05035ec22bdc082deb9a7a9c6e33f34b2d1881#diff-b7412e7e00778b956edd9dc6cc704c47R165